README.md

```diff
@@ -325,13 +325,13 @@ jobs:
             gen/ocm/ctf
 ```
 
-### Using components.yaml
+### Using component-constructor.yaml
```

Comment:
this seems redundant. The section above already references `component-constructor.yaml`.